hoverxref-intersphinx.rst

```diff
@@ -0,0 +1,114 @@
+can be distracting, since opening them might need either a context switch
+or keeping a new browser tab for a later moment.
+
+Here is where `sphinx-hoverxref`_ comes to the rescue:
+it scans the cross-references and attaches floating windows to them
+(either *tooltips* or *modal* dialogues)
+that embed the contents of the page the xrefs point to.
+This way, when the user is interested in a particular topic,
+they can hover the mouse over the link
+and display the floating window
+instead of opening the link.
+
+Moreover, the floating windows created by `sphinx-hoverxref`_
+also work if the embedded content requires a particular extension to be rendered.
+This makes it possible to embed content that includes mathematical equations, for example.
+
+Support for ``intersphinx``
+---------------------------
+
+Version 0.6b1 of `sphinx-hoverxref`_ introduces opt-in support for Intersphinx linking.
+`sphinx.ext.intersphinx <https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html>`_
+is a Sphinx extension
+that generates automatic links to the documentation of objects in other projects,
+and now it's possible to show floating windows on them too.
+
+.. figure:: img/sphinx-hoverxref-intersphinx.gif
+   :width: 80%
+   :align: center
+   :alt: sphinx-hoverxref displaying a modal window embedding an external project
+   :target: _images/sphinx-hoverxref-intersphinx.gif
+
+   sphinx-hoverxref displaying a modal window embedding an external project
+
+To enable it, the user needs to specify
+which projects from the `intersphinx_mapping <https://www.sphinx-doc.org/en/master/usage/extensions/intersphinx.html#confval-intersphinx_mapping>`_
+will be scanned by ``sphinx-hoverxref``, using a new configuration value:
+
+.. code-block:: python
+
+   hoverxref_intersphinx = [
+      "astropy",
+      "numpy",
+      "scipy",
+      "matplotlib",
+   ]
+
+Moreover, the look & feel of the current project is preserved
+even if the target documentation has a different theme or colors,
+which makes the result more visually consistent.
+
+.. note::
+
+   At the moment, the target project needs to be hosted on Read the Docs,
+   which is a limitation that we hope to remove in the future.
+
+You can draw inspiration from some community projects that are using this feature already:
+
+- `The Grill`_, an extension for digital content creation with Maya, Houdini, and other 3D modeling applications,
+  and references another project called `naming <https://naming.readthedocs.io/>`_.
+- `poliastro`_, a Python library for interactive Astrodynamics
+  that makes cross-references to `Astropy <https://docs.astropy.org/>`_ and other projects.
+
+We are excited to see what our users can do with `sphinx-hoverxref`_,
+and will continue working on it in the future.
+
+Considering using Read the Docs for your next Sphinx or MkDocs project?
+Check out `our documentation <https://docs.readthedocs.io/>`_ to get started!
```

Comment:
It's a small thing, but I love that we have this in every blog post :)

It should probably be part of the blog theme, and on the bottom of every page automatically in the future.

Reply:
😊  Yes, it feels like a great call to action!

> It should probably be part of the blog theme, and on the bottom of every page automatically in the future.

Yeah, I was thinking about it yesterday! Somehow I wouldn't want to fight against it if I don't want to show it for a particular blog post. But I think it would be generally a good idea.